api/src/services/replication-limit-log.js

```diff
@@ -0,0 +1,93 @@
+const moment = require('moment');
+const db = require('../db');
+const logger = require('../logger');
+
+const LOG_TYPES = {
+  replicationCount: 'replication-count-'
+};
```

Comment:
We only have one type right now, so we don't need to have an Object.

Reply:
Fixed